tests/test_pluvo.py

```diff
@@ -552,14 +552,17 @@ def test_pluvo_get_progress_report(mocker):
     p = pluvo.Pluvo(token='token')
     mocker.patch.object(p, '_get_multiple')
 
-    retval = p.get_progress_report([1, 2], [3, 4], ['-student_id'], 10, 0)
+    retval = p.get_progress_report(
+        [1, 2], [3, 4], ['-student_id'], 10, 0, 5, 6)
```

Comment:
Use dates(string?) instead of 5 and 6.

Reply:
It doesn't really matter much for this test. All this is testing is that the function passes along the arguments to the request properly. Nothing is actually done with the content of the arguments.